sdks/python/apache_beam/io/iobase.py

```diff
@@ -1573,15 +1581,28 @@ def is_bounded(self):
     return True
 
 
+class _RestrictionCoder(coders.Coder):
+  def decode(self, value):
+    return _SDFBoundedSourceRestriction(SourceBundle(*pickler.loads(value)))
+
+  def encode(self, restriction):
+    return pickler.dumps((
```

Comment:
Should we be using dill/pickler here to avoid regressions?

Reply:
okay - we're currently using pickler. does that lg?